bin/run.sh

```diff
@@ -37,7 +37,12 @@ cd "${solution_dir}"
 # Run the tests for the provided implementation file and redirect stdout and
 # stderr to capture it
 test_file=$(jq -r '.files.test[0]' "${solution_dir}/.meta/config.json")
-test_output=$(wine cmd /c "${test_file}" 2>&1)
+
+# Run wine with block networking to prevent the tests from making network calls
+test_output=$(sudo unshare -n runuser fabian -c "wine cmd /c \"${test_file}\"" 2>&1)
```

Comment:
This doesn't work in docker, as there is no `sudo` command. Can this be done in the `Dockerfile` instead somehow?

Reply:
Ah right, I didn't realize it because I couldn't try it, I found a resource like this, maybe it can really solve our problem, that is, it can speed up the process. https://superuser.com/questions/249962/how-can-i-disable-internet-access-for-programs-running-in-wine
 
By the way, let's not forget that the batch language is already a bit cumbersome.

Reply:
I haven't had any luck with that yet

Reply:
We close the network when run docker, shouldn't wine's requests be rejected anyway?

Reply:
Yes but it seems to hang. There is something that I can try

Reply:
> Yes but it seems to hang. There is something that I can try

How did your trials turn out?

Reply:
I managed to fix it using something we use for a couple of other test runners too: using an internal network